<ui version="4.0" >
 <class>CRecordDialogContent</class>
 <widget class="QWidget" name="CRecordDialogContent" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="cuePrefixEdit" />
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="cueNumberSpinBox" >
            <property name="minimum" >
             <number>-1</number>
            </property>
            <property name="maximum" >
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Cue Number:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Cue Prefix:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2" >
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="addCueButton" >
              <property name="text" >
               <string>Add Cue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addAnchoredCueButton" >
              <property name="text" >
               <string>Add Anchored Cue</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="1" column="1" >
           <widget class="QLabel" name="waitingForThresholdLED" >
            <property name="text" >
             <string>O</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="recordingLED" >
            <property name="text" >
             <string>O</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Waiting for Threshold:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Recording:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_5" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Record Length:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Record Size:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="recordLimitLabel" >
            <property name="text" >
             <string>Record Limit:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="recordedLengthStatusLabel" >
            <property name="text" >
             <string>00:00.000s</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="recordedSizeStatusLabel" >
            <property name="text" >
             <string>0b</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLabel" name="recordLimitStatusLabel" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="0" column="1" >
           <widget class="QLabel" name="DCOffsetLabel" >
            <property name="text" >
             <string>-inf dBFS</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_17" >
            <property name="text" >
             <string>DC Offset:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="QPushButton" name="DCOffsetCompensateButton" >
            <property name="text" >
             <string>Compensate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="resetClipCountButton" >
          <property name="text" >
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="clipCountLabel" >
          <property name="text" >
           <string>Clip Count: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="metersFrame" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="startButton" >
          <property name="text" >
           <string>Record/Resume</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton" >
          <property name="text" >
           <string>Stop/Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="redoButton" >
          <property name="text" >
           <string>Redo/Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <widget class="QCheckBox" name="limitDurationCheckBox" >
          <property name="text" >
           <string>Limit Duration to</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" >
         <widget class="QCheckBox" name="beginOnThresholdCheckBox" >
          <property name="text" >
           <string>Begin on Threshold</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="CDurationSpinBox" name="durationSpinBox" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="accelerated" >
           <bool>true</bool>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>2147483647</number>
          </property>
          <property name="value" >
           <number>60000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QDoubleSpinBox" name="thresholdSpinBox" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="accelerated" >
           <bool>true</bool>
          </property>
          <property name="prefix" >
           <string/>
          </property>
          <property name="suffix" >
           <string>dbFS</string>
          </property>
          <property name="minimum" >
           <double>-200.000000000000000</double>
          </property>
          <property name="maximum" >
           <double>0.000000000000000</double>
          </property>
          <property name="value" >
           <double>-24.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDurationSpinBox</class>
   <extends>QSpinBox</extends>
   <header>CDurationSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>limitDurationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>durationSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>215</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel" >
     <x>347</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
